src/Idris/Package.idr

```diff
@@ -365,7 +366,11 @@ install pkg opts -- not used but might be in the future
          -- Make the package installation directory
          let installPrefix = prefix_dir (dirs (options defs)) </>
                              "idris2-" ++ showVersion False version
-         True <- coreLift $ changeDir installPrefix
+         maybeDestdir <- coreLift $ idrisGetEnv "DESTDIR"
+         let installPath = case maybeDestdir of
+              Just destdir => destdir </> installPrefix
+              Nothing => installPrefix
+         True <- coreLift $ changeDir installPath
              | False => throw (InternalError ("Can't change directory to " ++ installPrefix))
```

Comment:
Here the error message still refers to `installPrefix`.
Rather than naming the modified path `installPath`, you may want to shadow `installPrefix` tbh.

Reply:
that's mirror thing... also I guess CI will fail for now because bootstrapping version has no this changes